modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -178,12 +176,6 @@ private void _validateExternalReferenceCode(
 		String externalReferenceCode, long companyId, long listTypeDefinitionId,
 		long listTypeEntryId) {
 
-		if (Validator.isNull(externalReferenceCode) ||
-			!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-168886"))) {
-
-			return;
-		}
```

Comment:
@murilostodolni should we keep the `Validator.isNull(externalReferenceCode)` here?

Reply:
Yes @carolmariaabb! Just did it. Thank you